### ISSUE TITLE
post_detail.html: meta_description/meta_keywords vs. base.html:meta

### ISSUE THREAD

**Issue by fvgoto** [nephila/djangocms-blog#355]
What's about [post_detail.html#L4f](https://github.com/nephila/djangocms-blog/blob/develop/djangocms_blog/templates/djangocms_blog/post_detail.html#L4)?:
```
{% block meta_description %}{{ post.meta_description }}{% endblock meta_description %}
{% block meta_keywords %}{{ post.meta_keywords }}{% endblock meta_keywords %}
```
What's the idea of that and how is it intended to be "used"?

There's also `{% block meta %} ... {% include "meta/meta.html" %}` in [base.html#L3ff](https://github.com/nephila/djangocms-blog/blob/develop/djangocms_blog/templates/djangocms_blog/base.html#L3).

By including `{% block meta %}{% endblock meta %}` in my page skeleton template, meta tags are rendered correctly, but the blocks of `post_detail.html` are pretty useless.

Whereas when including the two individual blocks instead of the `meta` one, the block in `base.html` is useless (except for further meta like social media stuff). But, big point, the contents of the two individual blocks is just the meta content payload, without `<meta` tag and `name` attr...

I guess, this should be cleaned up somehow...
...resolve this duplication.
Whereas the blocks from `post_detail.html` don't work just like that. If, then they should maybe read something like this:
```
{% block meta_description %}
    <meta name="description" content="{{ post.meta_description }}" />
{% endblock meta_description %} 
```

PS:
I stepped into this because I wanted to include the tags into the meta-keywords as well (having something of this form:
```
{% block meta_keywords %}
    <meta name="keywords" content="{{ post.meta_keywords }},{{ post.tags.names|join:',' }}" />
{% endblock meta_keywords %} 
```
(the hardcoded ',' in between is not very clean...)
If (as a solution to this issue) the two concerned lines from `post_detail.html` were removed in the project repo, it would be harder for the user to come up with a solution to my problem-case depicted above...

**Comment by yakky** [comment]
@fvgoto thanks for reporting that code. It's actually dead template code, unless you have corresponding blocks in the page template that those blocks override.
As it's just template code, you can easily customize it to suit your needs. A python-level solution to allow tags into keywords metadata (not that keywords are useful anymore..) will be far too intrusive (it will need a setting just for a limited use case ...)

**Comment by yakky** [comment]
Useless template code removed in #361 released in 0.8.13
thanks @fvgoto for reporting
